tensorboard/webapp/plugins/npmi/npmi_container.ts

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {select, Store} from '@ngrx/store';
+
+import {State as CoreState} from '../../core/store/core_types';
+import {getRunSelection} from '../../core/store/core_selectors';
+
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+
+@Component({
+  selector: 'npmi',
+  template: `
+    <npmi-component [runs]="runs$ | async"></npmi-component>
+  `,
+})
+export class NpmiContainer {
+  readonly runs$ = this.coreStore.pipe(select(getRunSelection));
+
+  constructor(private readonly coreStore: Store<CoreState>) {}
```

Comment:
`coreStore` -> `store`.  Also instead of typing it as `CoreState`, use `app_state.ts`. 

Rationale: (1) if you use some other selector in the future that accesses different state than `CoreState`, you will need to fix name and typing and (2) view should not care/know where the data is coming from.

Reply:
Created a app_state.ts that represents the state. Will later unify app state and core state.